### ISSUE TITLE
Design Enhancements for FrameTagger: UI/UX Improvements for Extension Window

### ISSUE THREAD

**Issue by umeshSinghVerma** [umeshSinghVerma/Youtube-Notes#2]
### Design Enhancements for FrameTagger: Improve UI/UX of the Extension Window 🎨

**Task**: Improve the overall design and user experience of the FrameTagger extension window. The current layout needs visual enhancements to make it more intuitive, user-friendly, and aligned with modern design principles.

**Requirements**:
- Follow a clean and minimalistic design approach, ensuring ease of use for the core features (capturing screenshots, annotations, and organizing snapshots).
- Update the color scheme to be visually appealing while maintaining accessibility standards (e.g., contrast for readability).
- Improve the layout to make the extension’s controls (buttons, icons) more intuitive and accessible.

**Additional Notes**:
- Consider adding animations or transitions to enhance the user experience without impacting performance.
- Propose design mockups before final implementation for review.

**Comment by Thunder-Blaze** [comment]
Sir can you please assign this to me
I'll try to complete within a week

**Comment by umeshSinghVerma** [comment]
"Thanks for your interest! 🎉 I've assigned the issue to you—excited to see your contribution! 🙌

If you haven't yet, please star ⭐ the repo to support FrameTagger! If you get stuck anywhere, feel free to reach out. 😊"

**Comment by umeshSinghVerma** [comment]
Yes, these mockups are really cool! 🎨 However, let’s aim for something that aligns even more closely with YouTube’s design language. Imagine you’re designing not just the extension, but YouTube itself—none of the current design elements like the red-colored logo or other features need to stay the same. You have full creative freedom to redesign everything in your own way. The goal is to make it feel like this is an original feature that YouTube’s own developers created.

Looking forward to seeing what you come up with! 😊

**Comment by umeshSinghVerma** [comment]
Hi @Thunder-Blaze! This new mockup looks great, and I see you've also integrated the delete button, which is a nice touch. I'll discuss it with the team and get back to you with our feedback soon. 😊

Thanks for your hard work so far!

**Comment by Thunder-Blaze** [comment]
ok sir 🫡
